examples/resources/umaa_521/UMAA/EO/AnchorStatus/AnchorReportType.idl

```diff
@@ -41,7 +41,7 @@ module UMAA
       struct AnchorReportType
       {
       // The current length of chain/rope that is paid out.
-      UMAA::Common::Measurement::Distance ropeLengthPaidOut;
+      UMAA::Common::Measurement::Distance rodeLengthPaidOut;

```

Comment:
`//TODO` to show the typo

Reply:
These are UMAA defined, outside of our control